src/views/Editor/index.tsx

```diff
@@ -35,7 +35,7 @@ export const Editor = (props: ViewProps): JSX.Element => {
     return new HocuspocusProvider({
       websocketProvider: hocuspocusWebsocket,
       name: documentId,
-      token: jwt.accessToken as string,
+      token: jwt.access_token as string,
```

Comment:
`access_token` falls under `JWTPayload`s catch all typing `[propName: string]: unknown`. Should probably be added to our `JWT` type, then we dont have to cast it as string.

Reply:
True!